mlir/include/mlir/Dialect/AMDGPU/IR/AMDGPU.td

```diff
@@ -437,6 +437,11 @@ def AMDGPU_DPPOp : AMDGPU_Op<"dpp", [SameTypeOperands, AllTypesMatch<["result",
   let summary = "AMDGPU DPP operation";
   let description = [{
     This operation represents DPP functionality in a GPU program.
+
+    The behavior should be equivalent to:
```

Comment:
This is not very clear to me.. What about using some English wording to explain too?

Reply:
While on the one hand, this is a low-level operation one might want to crack open the ISA manual for, I think we could also roughly specify the behavior here like you said.

Something like

This operation will result in the values from the `src` argument, permuted according to the DPP attributes (the permutation, row mask, and bounds control arguments). If a lane is masked off or has no value to receive, this operation will instead produce the value of `old`.